test/intentBuilder.test.ts

```diff
@@ -34,18 +35,20 @@ describe('execute function use cases tests', () => {
 
     // Check the balance after faucet
     const balanceAfter = await checkBalance(senderAddress);
-    expect(parseFloat(balanceAfter)).toBe(0.5);
+    console.log('balanceAfter', balanceAfter);
+    expect(parseFloat(balanceAfter)).toBe(1); // 1ETH fueled
   }, 100000);
 
   it('ETH -> DAI Swap', async () => {
     const from = new Asset({
         address: TOKENS.ETH,
-        amount: toBigInt(0.1),
+        amount: toBigInt(1000000000000000),
         chainId: toBigInt(CHAINS.Ethereum),
       }),
       to = new Asset({
         address: TOKENS.Dai,
         chainId: toBigInt(CHAINS.Ethereum),
+        amount: toBigInt(1000000000000000),
```

Comment:
this field is mandatory now, the price input from the client helps to determine the slippage and wrongly input will fallback to default by the network.